### PR TITLE
Corrected HS60 v2 keyboard name in info.json

### PR DESCRIPTION
**Pull request by stanrc85**
## Description
I noticed the keyboard name and maintainer were incorrect in the HS60v2 info.json file, they have been updated now.  Let me know if your standard for "versions" are different, HS60V2 or HS60v2, or something else. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hs60/v2/info.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "keyboard_name": "clueboard/60",
-  "maintainer": "skullydazed",
+  "keyboard_name": "HS60v2",
+  "maintainer": "qmk",
```
Comment (yiancar) [inline keyboards/hs60/v2/info.json]: Could we change to "maintainer": "yiancar",

Comment (yiancar) [inline keyboards/hs60/v2/info.json]: Could we change this to: `"maintainer": "yiancar",` please?:)

**Comment by yiancar** [comment]
Thank for noticing this and putting up a PR:)

File: keyboards/hs60/v2/info.json (hunk 1 above)
Comment (stanrc85) [reply keyboards/hs60/v2/info.json]: Done, now I know who to come to with my questions :)

**Comment by yiancar** [comment]
+1 from me!

**Pull request merged**